proxy/src/main/java/com/velocitypowered/proxy/connection/client/ConnectedPlayer.java

```diff
@@ -695,8 +695,12 @@ void teardown() {
 
     DisconnectEvent.LoginStatus status;
     if (connectedPlayer.isPresent()) {
-      status = connectedPlayer.get() == this ? LoginStatus.SUCCESSFUL_LOGIN
-          : LoginStatus.CONFLICTING_LOGIN;
+      if (!connectedPlayer.get().getCurrentServer().isPresent()) {
+        status = LoginStatus.PRE_SERVER_JOIN;
+      } else {
+        status = connectedPlayer.get() == this ? LoginStatus.SUCCESSFUL_LOGIN
```

Comment:
See L698

Reply:
Fixed in 4adc859cf11c62319f637d090bf0bc8d11f23ae8